File: season_1/programming_lab/TDE_01/exercise_05.c
#include <stdio.h>

int main() {
    int numero;

    printf("Digite um número: ");

    if (scanf("%d", &numero) != 1) {
        printf("Erro: voce nao digitou um numero inteiro valido.\n");
        return 1;
    }

    if (numero % 3 == 0 && numero % 7 == 0) {
        printf("O numero e divisivel por 3 e por 7.\n");
    } else {
        printf("O numero NAO e divisivel por 3 e por 7.\n");
    }

    return 0;
}

File: exercise_05.py
try:
    numero = int(input("Digite um número: "))

    if numero % 3 == 0 and numero % 7 == 0:
        print("O número é divisível por 3 e por 7.")
    else:
        print("O número NÃO é divisível por 3 e por 7.")

except ValueError:
    print("Erro: você não digitou um número inteiro válido.")
